frontend/src/features/workspace/components/EmptyWorkspace/EmptyWorkspace.tsx

```diff
@@ -22,9 +22,10 @@ export const EmptyWorkspace = ({
 
   return (
     <Flex
-      justify="center"
+      justify="flex-start"
       flexDir="column"
       align="center"
+      h="100vh"
```

Comment:
should we use the `css` prop instead? `100vh` SUCKS on iOS safari.

css utils are in `frontend/src/utils/fillHeightCss.ts`

Reply:
okehh [3b2368f](https://github.com/opengovsg/FormSG/pull/4964/commits/3b2368f41f136d270e6baa6b02d9a711acb16288)